src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2824,45 +2823,6 @@ private bool IsMisplacedModifier(SyntaxListBuilder modifiers, SyntaxList<Attribu
             return false;
         }
 
-        private static bool ContainsErrorDiagnostic(GreenNode node)
-        {
-            // ContainsDiagnostics returns true if this node (or any descendants) contain any sort of error.  However,
-            // GetDiagnostics() only returns diagnostics at that node itself.  So we have to explicitly walk down the
-            // tree to find out if the diagnostics are error or not.
-
-            // Quick check to avoid any unnecessary work.
-            if (node.ContainsDiagnostics)
-            {
-                var stack = ArrayBuilder<GreenNode>.GetInstance();
-                try
-                {
-                    stack.Push(node);
-
-                    while (stack.Count > 0)
-                    {
-                        var current = stack.Pop();
-                        if (!current.ContainsDiagnostics)
-                            continue;
```

Comment:
>block [](http://example.com/codeflow?start=16&length=5)

nit: consider renaming `block` to `tryBlock`

Reply:
done.